app/assets/stylesheets/theme/custom.css.scss

```diff
@@ -41,4 +41,42 @@ select.form-control.search_options {
     padding: 5px;
     width: 138px;
   }
+}
+
+// pagination
+.pagination a, .pagination span.current, .pagination span.gap {
+  float: left;
+  padding: 0 15px;
+  line-height: 38px;
+  text-decoration: none;
+  background-color: white;
+  border-left-width: 0;
+}
+
+.pagination {
+  .first{
+    padding : 0;
+    float: none;
+    border: none;
+  }
+  .prev {
+    padding : 0;
+    float: none;
+    border: none;
+  }
+  .page{
+    padding : 0;
```

Comment:
Properties should be ordered border, float, padding<br>Property name should be immediately followed by a colon